### ISSUE TITLE
Any way to get the expected training time written to a file?

### ISSUE THREAD

**Issue by mon28** [AlexeyAB/darknet#6661]
Hi,

If during the training, we have set the mjpeg_port, we are able to see the training graph along with the expected training time left at the bottom right.
 Is there any way to export and keep updating this data in a file while the training is going on?

@AlexeyAB

**Comment by AlexeyAB** [comment]
* Locally: This images is stored to the `chart.png` file automatically

* Remotely: You can open URL in web-browser, then click right-mouse-button on the image in Web-browser and press `save as ...` so training progress will be stored as mjpeg-video.

**Comment by mon28** [comment]
Yes i am aware of this. What i need is some way of accessing this data texually so that it can be parsed by a program.

**Comment by johnny-mueller** [comment]
@mon28 yes you can use tee to log the data.
It would look something like this and then saves the output during training in the file "log.txt":
```python
./darknet detector -map -dont_show -gpus 0,1 train cfg/coco.data cfg/yolov3.cfg 2>&1 | tee --append log.txt
```

**Comment by mon28** [comment]
Thanks @johnny-mueller
